### PR TITLE
Add a specific note about where to put iptables in /rw/config/

### PR DESCRIPTION
**Pull request by clayt0nk**
This would have probably saved me a fair bit of flailing around.

### CODE REVIEW

**Comment by andrewdavidwong** [comment]
Might be worth explaining that all commands in `/rw/config/rc.local` will b executed as root whenever the VM is started.

@marmarek

**Comment by clayt0nk** [comment]
It would in fact also be nice to explain what is special about qubes_firewall_user_script, to distinguish it from rc.local. I am guessing it gets re-run every time the Qubes firewall dynamically modifies it's firewall? Which process is why iptables settings in rc.local do not persist?

File: security/qubes-firewall.md [hunk 1]
```diff
@@ -315,3 +315,11 @@ fi
...
 This time testing should allow connectivity to the service as long as the
 service is up :-)
+
+Where to put firewall rules
+---------------------------
+
+Implicit in the above example, but worth calling attention to: for all 
+VMs EXCEPT proxy VMs, iptables commands should be added to the 
+'/rw/config/rc.local' script. For proxy VMs, iptables commands should 
+be added to '/rw/config/qubes_firewall_user_script'.
```
Comment (marmarek) [inline security/qubes-firewall.md]: `/rw/config/qubes-firewall-user-script`
Also worth adding a link to https://www.qubes-os.org/doc/config-files/

**Comment by clayt0nk** [comment]
Good suggestions all around, guys. I copied-n-pasted that script name, so there are multiple locations. I grepped the entire repo and hopefully got all of them. Hope I got that internal link right?

On a tangential note, from contributing.md:

	"Hard wrap Markdown lines at 80 characters."

I am resisting the urge to hard wrap every para. I touch to keep the diff clean, would you prefer I continue to do that or not?

**Comment by andrewdavidwong** [comment]
> I am resisting the urge to hard wrap every para.

I know that urge well. :)

> I touch to keep the diff clean, would you prefer I continue to do that or not?

It probably won't matter much in this case, since diff tools should be able to distinguish content changes from rewrapping. However, it would certainly be better to rewrap unrelated lines in a different commit, and generally I find it cleaner to do that sort of thing in a separate PR. For such a small file, however, that would probably be overkill. (If you decided to rewrap a lot of text across multiple files, though, a separate PR would be great.)

**Comment by clayt0nk** [comment]
Cool, I just made a hard-wrap-only commit of config-files.md, the other files I touched were actually in pretty good shape in this department.

**Pull request merged**